### PR TITLE
Plain js with objects

### PR DESCRIPTION
**Pull request by Thoth1111**
In this branch, we built code to allow for the addition and deletion of books, and updating of the local storage in light of the changes. We achieved this through the following steps:
- Added .gitignore and linter files to the project
- Built up a basic html page
- Added  and linked a script file (index.js)
- Created a working functions for the addition of books to the collection
- Created an eventlistener to load and display the books from the storage when the page is reloaded
- Created a working function to remove books from the collection and update the local storage accordingly.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+const title = document.querySelector('.title_input');
+const author = document.querySelector('.author_input');
+const empty = document.querySelector('.empty_bookstore');
+const form = document.querySelector('.book_form');
+const tableBody = document.querySelector('.table_body');
+const storedBooks = JSON.parse(localStorage.getItem('bookstored'));
```
Comment (julie-ify) [inline index.js]: - [x] See the error I get when I try to add a book for the first time. You probably need to clear your local storage to replicate this error.

  image

  You can fix this by initializing `storedBooks` to an empty array in the event it is `null`:

  ```
  const storedBooks = JSON.parse(localStorage.getItem('bookstored')) || [];
  ```

File: index.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  }
+  tableBody.innerHTML = eachTableRow;
+  const deleteBookBtn = document.querySelectorAll('.remove_btn');
+  deleteBookBtn.forEach((elementBtn) => {
+    elementBtn.addEventListener('click', () => {
+      const index = elementBtn.getAttribute('id');
+      storedBooks.splice(index, 1);
+      /* eslint-disable */
+      localStorage.clear;
+      /* eslint-enable */
```
Comment (julie-ify) [inline index.js]: - [x] **[OPTIONAL]** Do we really need to clear the local storage every time? The `setItem()` typically method overrides whatever exists in the `storedBooks` variable so there is no need to clear the local storage.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @Thoth1111 and @EmekaEmmanuel,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (Thoth1111) [reply index.js]: Thank you @julie-ify. This went unnoticed but we have seen the error and addressed it.

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Thoth1111 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

Good feature branch name ✔️
No linter errors ✔️
Descriptive pull request ✔️
Descriptive commit messages ✔️
Good code organization and working app ✔️
Descriptive README ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**